# Logfile created on 2014-07-10 14:11:03 by logger.rb/44203
I, [2014-07-10T14:11:03.330700 #16774]  INFO -- : *** below you find the most recent exception thrown, this will be likely (but not certainly) the exception that made the application exit abnormally ***
E, [2014-07-10T14:11:03.330847 #16774] ERROR -- : Must be one of [:method, :headers, :body, :heartbeat] (AMQ::Protocol::FrameTypeError)
/home/yuva/.rvm/gems/ruby-2.1.1/gems/amq-protocol-1.9.2/lib/amq/protocol/frame.rb:64:in `decode_header'
/home/yuva/.rvm/gems/ruby-2.1.1/gems/bunny-1.2.1/lib/bunny/transport.rb:208:in `read_next_frame'
/home/yuva/.rvm/gems/ruby-2.1.1/gems/bunny-1.2.1/lib/bunny/session.rb:923:in `init_connection'
/home/yuva/.rvm/gems/ruby-2.1.1/gems/bunny-1.2.1/lib/bunny/session.rb:254:in `start'
/home/yuva/Desktop/arvind/peatio/peatio-master/lib/daemons/amqp_daemon.rb:17:in `<top (required)>'
/home/yuva/.rvm/gems/ruby-2.1.1/gems/daemons-1.1.9/lib/daemons/application.rb:203:in `load'
/home/yuva/.rvm/gems/ruby-2.1.1/gems/daemons-1.1.9/lib/daemons/application.rb:203:in `start_load'
/home/yuva/.rvm/gems/ruby-2.1.1/gems/daemons-1.1.9/lib/daemons/application.rb:298:in `start'
/home/yuva/.rvm/gems/ruby-2.1.1/gems/daemons-1.1.9/lib/daemons/controller.rb:70:in `run'
/home/yuva/.rvm/gems/ruby-2.1.1/gems/daemons-1.1.9/lib/daemons.rb:147:in `block in run'
/home/yuva/.rvm/gems/ruby-2.1.1/gems/daemons-1.1.9/lib/daemons/cmdline.rb:109:in `call'
/home/yuva/.rvm/gems/ruby-2.1.1/gems/daemons-1.1.9/lib/daemons/cmdline.rb:109:in `catch_exceptions'
/home/yuva/.rvm/gems/ruby-2.1.1/gems/daemons-1.1.9/lib/daemons.rb:146:in `run'
/home/yuva/.rvm/gems/ruby-2.1.1/gems/daemons-rails-1.2.1/lib/daemons/rails.rb:16:in `run'
lib/daemons/matching_ctl:11:in `<main>'
I, [2014-07-10T14:11:03.332759 #16774]  INFO -- : *** below you find all exception objects found in memory, some of them may have been thrown in your application, others may just be in memory because they are standard exceptions ***
E, [2014-07-10T14:11:03.332862 #16774] ERROR -- : exception reentered (fatal)

E, [2014-07-10T14:11:03.339756 #16774] ERROR -- : stream closed (IOError)

E, [2014-07-10T14:11:03.339903 #16774] ERROR -- : failed to allocate memory (NoMemoryError)

E, [2014-07-10T14:11:03.339971 #16774] ERROR -- : stack level too deep (SystemStackError)

E, [2014-07-10T14:11:03.374238 #16774] ERROR -- : Value will be available once the pipeline executes. (Redis::FutureNotReady)

E, [2014-07-10T14:11:03.539123 #16774] ERROR -- : no such frame (ArgumentError)
/home/yuva/.rvm/gems/ruby-2.1.1/gems/binding_of_caller-0.7.2/lib/binding_of_caller/mri2.rb:25:in `frame_binding'
/home/yuva/.rvm/gems/ruby-2.1.1/gems/binding_of_caller-0.7.2/lib/binding_of_caller/mri2.rb:25:in `block (2 levels) in callers'
/home/yuva/.rvm/gems/ruby-2.1.1/gems/binding_of_caller-0.7.2/lib/binding_of_caller/mri2.rb:23:in `loop'
/home/yuva/.rvm/gems/ruby-2.1.1/gems/binding_of_caller-0.7.2/lib/binding_of_caller/mri2.rb:23:in `block in callers'
/home/yuva/.rvm/gems/ruby-2.1.1/gems/binding_of_caller-0.7.2/lib/binding_of_caller/mri2.rb:21:in `open'
/home/yuva/.rvm/gems/ruby-2.1.1/gems/binding_of_caller-0.7.2/lib/binding_of_caller/mri2.rb:21:in `callers'
/home/yuva/.rvm/gems/ruby-2.1.1/gems/better_errors-1.1.0/lib/better_errors/core_ext/exception.rb:9:in `block in <class:Exception>'
/home/yuva/.rvm/rubies/ruby-2.1.1/lib/ruby/2.1.0/open-uri.rb:36:in `initialize'
/home/yuva/.rvm/rubies/ruby-2.1.1/lib/ruby/2.1.0/open-uri.rb:36:in `open'
/home/yuva/.rvm/rubies/ruby-2.1.1/lib/ruby/2.1.0/open-uri.rb:36:in `open'
/home/yuva/.rvm/rubies/ruby-2.1.1/lib/ruby/2.1.0/logger.rb:592:in `open_logfile'
/home/yuva/.rvm/rubies/ruby-2.1.1/lib/ruby/2.1.0/logger.rb:549:in `initialize'
/home/yuva/.rvm/rubies/ruby-2.1.1/lib/ruby/2.1.0/logger.rb:318:in `new'
/home/yuva/.rvm/rubies/ruby-2.1.1/lib/ruby/2.1.0/logger.rb:318:in `initialize'
/home/yuva/.rvm/gems/ruby-2.1.1/gems/daemons-1.1.9/lib/daemons/application.rb:353:in `new'
/home/yuva/.rvm/gems/ruby-2.1.1/gems/daemons-1.1.9/lib/daemons/application.rb:353:in `exception_log'
/home/yuva/.rvm/gems/ruby-2.1.1/gems/daemons-1.1.9/lib/daemons/application.rb:165:in `block in start_load'
E, [2014-07-10T14:11:03.540289 #16774] ERROR -- : No such file or directory @ rb_sysopen - /home/yuva/Desktop/arvind/peatio/peatio-master/log/peatio:amqp:matching.log (Errno::ENOENT)
/home/yuva/.rvm/rubies/ruby-2.1.1/lib/ruby/2.1.0/open-uri.rb:36:in `initialize'
/home/yuva/.rvm/rubies/ruby-2.1.1/lib/ruby/2.1.0/open-uri.rb:36:in `open'
/home/yuva/.rvm/rubies/ruby-2.1.1/lib/ruby/2.1.0/open-uri.rb:36:in `open'
/home/yuva/.rvm/rubies/ruby-2.1.1/lib/ruby/2.1.0/logger.rb:592:in `open_logfile'
/home/yuva/.rvm/rubies/ruby-2.1.1/lib/ruby/2.1.0/logger.rb:549:in `initialize'
/home/yuva/.rvm/rubies/ruby-2.1.1/lib/ruby/2.1.0/logger.rb:318:in `new'
/home/yuva/.rvm/rubies/ruby-2.1.1/lib/ruby/2.1.0/logger.rb:318:in `initialize'
/home/yuva/.rvm/gems/ruby-2.1.1/gems/daemons-1.1.9/lib/daemons/application.rb:353:in `new'
/home/yuva/.rvm/gems/ruby-2.1.1/gems/daemons-1.1.9/lib/daemons/application.rb:353:in `exception_log'
/home/yuva/.rvm/gems/ruby-2.1.1/gems/daemons-1.1.9/lib/daemons/application.rb:165:in `block in start_load'
E, [2014-07-10T14:11:03.540395 #16774] ERROR -- : no such frame (ArgumentError)
/home/yuva/.rvm/gems/ruby-2.1.1/gems/binding_of_caller-0.7.2/lib/binding_of_caller/mri2.rb:25:in `frame_binding'
/home/yuva/.rvm/gems/ruby-2.1.1/gems/binding_of_caller-0.7.2/lib/binding_of_caller/mri2.rb:25:in `block (2 levels) in callers'
/home/yuva/.rvm/gems/ruby-2.1.1/gems/binding_of_caller-0.7.2/lib/binding_of_caller/mri2.rb:23:in `loop'
/home/yuva/.rvm/gems/ruby-2.1.1/gems/binding_of_caller-0.7.2/lib/binding_of_caller/mri2.rb:23:in `block in callers'
/home/yuva/.rvm/gems/ruby-2.1.1/gems/binding_of_caller-0.7.2/lib/binding_of_caller/mri2.rb:21:in `open'
/home/yuva/.rvm/gems/ruby-2.1.1/gems/binding_of_caller-0.7.2/lib/binding_of_caller/mri2.rb:21:in `callers'
/home/yuva/.rvm/gems/ruby-2.1.1/gems/better_errors-1.1.0/lib/better_errors/core_ext/exception.rb:9:in `block in <class:Exception>'
/home/yuva/.rvm/gems/ruby-2.1.1/gems/amq-protocol-1.9.2/lib/amq/protocol/frame.rb:64:in `decode_header'
/home/yuva/.rvm/gems/ruby-2.1.1/gems/bunny-1.2.1/lib/bunny/transport.rb:208:in `read_next_frame'
/home/yuva/.rvm/gems/ruby-2.1.1/gems/bunny-1.2.1/lib/bunny/session.rb:923:in `init_connection'
/home/yuva/.rvm/gems/ruby-2.1.1/gems/bunny-1.2.1/lib/bunny/session.rb:254:in `start'
/home/yuva/Desktop/arvind/peatio/peatio-master/lib/daemons/amqp_daemon.rb:17:in `<top (required)>'
/home/yuva/.rvm/gems/ruby-2.1.1/gems/daemons-1.1.9/lib/daemons/application.rb:203:in `load'
/home/yuva/.rvm/gems/ruby-2.1.1/gems/daemons-1.1.9/lib/daemons/application.rb:203:in `start_load'
/home/yuva/.rvm/gems/ruby-2.1.1/gems/daemons-1.1.9/lib/daemons/application.rb:298:in `start'
/home/yuva/.rvm/gems/ruby-2.1.1/gems/daemons-1.1.9/lib/daemons/controller.rb:70:in `run'
/home/yuva/.rvm/gems/ruby-2.1.1/gems/daemons-1.1.9/lib/daemons.rb:147:in `block in run'
/home/yuva/.rvm/gems/ruby-2.1.1/gems/daemons-1.1.9/lib/daemons/cmdline.rb:109:in `call'
/home/yuva/.rvm/gems/ruby-2.1.1/gems/daemons-1.1.9/lib/daemons/cmdline.rb:109:in `catch_exceptions'
/home/yuva/.rvm/gems/ruby-2.1.1/gems/daemons-1.1.9/lib/daemons.rb:146:in `run'
/home/yuva/.rvm/gems/ruby-2.1.1/gems/daemons-rails-1.2.1/lib/daemons/rails.rb:16:in `run'
lib/daemons/matching_ctl:11:in `<main>'
E, [2014-07-10T14:11:03.540484 #16774] ERROR -- : Must be one of [:method, :headers, :body, :heartbeat] (AMQ::Protocol::FrameTypeError)
/home/yuva/.rvm/gems/ruby-2.1.1/gems/amq-protocol-1.9.2/lib/amq/protocol/frame.rb:64:in `decode_header'
/home/yuva/.rvm/gems/ruby-2.1.1/gems/bunny-1.2.1/lib/bunny/transport.rb:208:in `read_next_frame'
/home/yuva/.rvm/gems/ruby-2.1.1/gems/bunny-1.2.1/lib/bunny/session.rb:923:in `init_connection'
/home/yuva/.rvm/gems/ruby-2.1.1/gems/bunny-1.2.1/lib/bunny/session.rb:254:in `start'
/home/yuva/Desktop/arvind/peatio/peatio-master/lib/daemons/amqp_daemon.rb:17:in `<top (required)>'
/home/yuva/.rvm/gems/ruby-2.1.1/gems/daemons-1.1.9/lib/daemons/application.rb:203:in `load'
/home/yuva/.rvm/gems/ruby-2.1.1/gems/daemons-1.1.9/lib/daemons/application.rb:203:in `start_load'
/home/yuva/.rvm/gems/ruby-2.1.1/gems/daemons-1.1.9/lib/daemons/application.rb:298:in `start'
/home/yuva/.rvm/gems/ruby-2.1.1/gems/daemons-1.1.9/lib/daemons/controller.rb:70:in `run'
/home/yuva/.rvm/gems/ruby-2.1.1/gems/daemons-1.1.9/lib/daemons.rb:147:in `block in run'
/home/yuva/.rvm/gems/ruby-2.1.1/gems/daemons-1.1.9/lib/daemons/cmdline.rb:109:in `call'
/home/yuva/.rvm/gems/ruby-2.1.1/gems/daemons-1.1.9/lib/daemons/cmdline.rb:109:in `catch_exceptions'
/home/yuva/.rvm/gems/ruby-2.1.1/gems/daemons-1.1.9/lib/daemons.rb:146:in `run'
/home/yuva/.rvm/gems/ruby-2.1.1/gems/daemons-rails-1.2.1/lib/daemons/rails.rb:16:in `run'
lib/daemons/matching_ctl:11:in `<main>'
I, [2014-07-14T17:53:58.079828 #25232]  INFO -- : *** below you find the most recent exception thrown, this will be likely (but not certainly) the exception that made the application exit abnormally ***
E, [2014-07-14T17:53:58.166038 #25232] ERROR -- : no such frame (ArgumentError)
/home/yuva/.rvm/gems/ruby-2.1.2/gems/binding_of_caller-0.7.2/lib/binding_of_caller/mri2.rb:25:in `frame_binding'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/binding_of_caller-0.7.2/lib/binding_of_caller/mri2.rb:25:in `block (2 levels) in callers'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/binding_of_caller-0.7.2/lib/binding_of_caller/mri2.rb:23:in `loop'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/binding_of_caller-0.7.2/lib/binding_of_caller/mri2.rb:23:in `block in callers'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/binding_of_caller-0.7.2/lib/binding_of_caller/mri2.rb:21:in `open'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/binding_of_caller-0.7.2/lib/binding_of_caller/mri2.rb:21:in `callers'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/better_errors-1.1.0/lib/better_errors/core_ext/exception.rb:9:in `block in <class:Exception>'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/bunny-1.2.1/lib/bunny/cruby/socket.rb:41:in `read_nonblock'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/bunny-1.2.1/lib/bunny/cruby/socket.rb:41:in `block in read_fully'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/bunny-1.2.1/lib/bunny/cruby/socket.rb:40:in `loop'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/bunny-1.2.1/lib/bunny/cruby/socket.rb:40:in `read_fully'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/bunny-1.2.1/lib/bunny/transport.rb:199:in `read_next_frame'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/bunny-1.2.1/lib/bunny/session.rb:990:in `open_connection'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/bunny-1.2.1/lib/bunny/session.rb:255:in `start'
/home/yuva/Desktop/arvind/peatio/peatio-master/lib/daemons/amqp_daemon.rb:17:in `<top (required)>'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/daemons-1.1.9/lib/daemons/application.rb:203:in `load'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/daemons-1.1.9/lib/daemons/application.rb:203:in `start_load'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/daemons-1.1.9/lib/daemons/application.rb:298:in `start'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/daemons-1.1.9/lib/daemons/controller.rb:70:in `run'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/daemons-1.1.9/lib/daemons.rb:147:in `block in run'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/daemons-1.1.9/lib/daemons/cmdline.rb:109:in `call'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/daemons-1.1.9/lib/daemons/cmdline.rb:109:in `catch_exceptions'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/daemons-1.1.9/lib/daemons.rb:146:in `run'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/daemons-rails-1.2.1/lib/daemons/rails.rb:16:in `run'
lib/daemons/matching_ctl:11:in `<main>'
I, [2014-07-14T17:53:58.166241 #25232]  INFO -- : *** below you find all exception objects found in memory, some of them may have been thrown in your application, others may just be in memory because they are standard exceptions ***
E, [2014-07-14T17:53:58.166388 #25232] ERROR -- : exception reentered (fatal)

E, [2014-07-14T17:53:58.252741 #25232] ERROR -- : Resource temporarily unavailable - read would block (IO::EAGAINWaitReadable)
/home/yuva/.rvm/gems/ruby-2.1.2/gems/bunny-1.2.1/lib/bunny/cruby/socket.rb:41:in `read_nonblock'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/bunny-1.2.1/lib/bunny/cruby/socket.rb:41:in `block in read_fully'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/bunny-1.2.1/lib/bunny/cruby/socket.rb:40:in `loop'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/bunny-1.2.1/lib/bunny/cruby/socket.rb:40:in `read_fully'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/bunny-1.2.1/lib/bunny/transport.rb:199:in `read_next_frame'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/bunny-1.2.1/lib/bunny/reader_loop.rb:63:in `run_once'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/bunny-1.2.1/lib/bunny/reader_loop.rb:33:in `block in run_loop'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/bunny-1.2.1/lib/bunny/reader_loop.rb:30:in `loop'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/bunny-1.2.1/lib/bunny/reader_loop.rb:30:in `run_loop'
E, [2014-07-14T17:53:58.253128 #25232] ERROR -- : no such frame (ArgumentError)
/home/yuva/.rvm/gems/ruby-2.1.2/gems/binding_of_caller-0.7.2/lib/binding_of_caller/mri2.rb:25:in `frame_binding'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/binding_of_caller-0.7.2/lib/binding_of_caller/mri2.rb:25:in `block (2 levels) in callers'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/binding_of_caller-0.7.2/lib/binding_of_caller/mri2.rb:23:in `loop'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/binding_of_caller-0.7.2/lib/binding_of_caller/mri2.rb:23:in `block in callers'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/binding_of_caller-0.7.2/lib/binding_of_caller/mri2.rb:21:in `open'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/binding_of_caller-0.7.2/lib/binding_of_caller/mri2.rb:21:in `callers'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/better_errors-1.1.0/lib/better_errors/core_ext/exception.rb:9:in `block in <class:Exception>'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/bunny-1.2.1/lib/bunny/cruby/socket.rb:41:in `read_nonblock'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/bunny-1.2.1/lib/bunny/cruby/socket.rb:41:in `block in read_fully'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/bunny-1.2.1/lib/bunny/cruby/socket.rb:40:in `loop'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/bunny-1.2.1/lib/bunny/cruby/socket.rb:40:in `read_fully'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/bunny-1.2.1/lib/bunny/transport.rb:199:in `read_next_frame'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/bunny-1.2.1/lib/bunny/reader_loop.rb:63:in `run_once'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/bunny-1.2.1/lib/bunny/reader_loop.rb:33:in `block in run_loop'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/bunny-1.2.1/lib/bunny/reader_loop.rb:30:in `loop'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/bunny-1.2.1/lib/bunny/reader_loop.rb:30:in `run_loop'
E, [2014-07-14T17:53:58.253454 #25232] ERROR -- : Resource temporarily unavailable - read would block (IO::EAGAINWaitReadable)
/home/yuva/.rvm/gems/ruby-2.1.2/gems/bunny-1.2.1/lib/bunny/cruby/socket.rb:41:in `read_nonblock'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/bunny-1.2.1/lib/bunny/cruby/socket.rb:41:in `block in read_fully'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/bunny-1.2.1/lib/bunny/cruby/socket.rb:40:in `loop'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/bunny-1.2.1/lib/bunny/cruby/socket.rb:40:in `read_fully'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/bunny-1.2.1/lib/bunny/transport.rb:199:in `read_next_frame'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/bunny-1.2.1/lib/bunny/reader_loop.rb:63:in `run_once'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/bunny-1.2.1/lib/bunny/reader_loop.rb:33:in `block in run_loop'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/bunny-1.2.1/lib/bunny/reader_loop.rb:30:in `loop'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/bunny-1.2.1/lib/bunny/reader_loop.rb:30:in `run_loop'
E, [2014-07-14T17:53:58.253822 #25232] ERROR -- : no such frame (ArgumentError)
/home/yuva/.rvm/gems/ruby-2.1.2/gems/binding_of_caller-0.7.2/lib/binding_of_caller/mri2.rb:25:in `frame_binding'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/binding_of_caller-0.7.2/lib/binding_of_caller/mri2.rb:25:in `block (2 levels) in callers'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/binding_of_caller-0.7.2/lib/binding_of_caller/mri2.rb:23:in `loop'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/binding_of_caller-0.7.2/lib/binding_of_caller/mri2.rb:23:in `block in callers'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/binding_of_caller-0.7.2/lib/binding_of_caller/mri2.rb:21:in `open'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/binding_of_caller-0.7.2/lib/binding_of_caller/mri2.rb:21:in `callers'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/better_errors-1.1.0/lib/better_errors/core_ext/exception.rb:9:in `block in <class:Exception>'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/bunny-1.2.1/lib/bunny/cruby/socket.rb:41:in `read_nonblock'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/bunny-1.2.1/lib/bunny/cruby/socket.rb:41:in `block in read_fully'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/bunny-1.2.1/lib/bunny/cruby/socket.rb:40:in `loop'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/bunny-1.2.1/lib/bunny/cruby/socket.rb:40:in `read_fully'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/bunny-1.2.1/lib/bunny/transport.rb:199:in `read_next_frame'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/bunny-1.2.1/lib/bunny/reader_loop.rb:63:in `run_once'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/bunny-1.2.1/lib/bunny/reader_loop.rb:33:in `block in run_loop'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/bunny-1.2.1/lib/bunny/reader_loop.rb:30:in `loop'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/bunny-1.2.1/lib/bunny/reader_loop.rb:30:in `run_loop'
E, [2014-07-14T17:53:58.253982 #25232] ERROR -- : Resource temporarily unavailable - read would block (IO::EAGAINWaitReadable)
/home/yuva/.rvm/gems/ruby-2.1.2/gems/bunny-1.2.1/lib/bunny/cruby/socket.rb:41:in `read_nonblock'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/bunny-1.2.1/lib/bunny/cruby/socket.rb:41:in `block in read_fully'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/bunny-1.2.1/lib/bunny/cruby/socket.rb:40:in `loop'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/bunny-1.2.1/lib/bunny/cruby/socket.rb:40:in `read_fully'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/bunny-1.2.1/lib/bunny/transport.rb:199:in `read_next_frame'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/bunny-1.2.1/lib/bunny/reader_loop.rb:63:in `run_once'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/bunny-1.2.1/lib/bunny/reader_loop.rb:33:in `block in run_loop'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/bunny-1.2.1/lib/bunny/reader_loop.rb:30:in `loop'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/bunny-1.2.1/lib/bunny/reader_loop.rb:30:in `run_loop'
E, [2014-07-14T17:53:58.254624 #25232] ERROR -- : no such frame (ArgumentError)
/home/yuva/.rvm/gems/ruby-2.1.2/gems/binding_of_caller-0.7.2/lib/binding_of_caller/mri2.rb:25:in `frame_binding'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/binding_of_caller-0.7.2/lib/binding_of_caller/mri2.rb:25:in `block (2 levels) in callers'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/binding_of_caller-0.7.2/lib/binding_of_caller/mri2.rb:23:in `loop'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/binding_of_caller-0.7.2/lib/binding_of_caller/mri2.rb:23:in `block in callers'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/binding_of_caller-0.7.2/lib/binding_of_caller/mri2.rb:21:in `open'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/binding_of_caller-0.7.2/lib/binding_of_caller/mri2.rb:21:in `callers'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/better_errors-1.1.0/lib/better_errors/core_ext/exception.rb:9:in `block in <class:Exception>'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/bunny-1.2.1/lib/bunny/cruby/socket.rb:41:in `read_nonblock'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/bunny-1.2.1/lib/bunny/cruby/socket.rb:41:in `block in read_fully'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/bunny-1.2.1/lib/bunny/cruby/socket.rb:40:in `loop'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/bunny-1.2.1/lib/bunny/cruby/socket.rb:40:in `read_fully'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/bunny-1.2.1/lib/bunny/transport.rb:199:in `read_next_frame'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/bunny-1.2.1/lib/bunny/reader_loop.rb:63:in `run_once'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/bunny-1.2.1/lib/bunny/reader_loop.rb:33:in `block in run_loop'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/bunny-1.2.1/lib/bunny/reader_loop.rb:30:in `loop'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/bunny-1.2.1/lib/bunny/reader_loop.rb:30:in `run_loop'
E, [2014-07-14T17:53:58.269938 #25232] ERROR -- : Resource temporarily unavailable - read would block (IO::EAGAINWaitReadable)
/home/yuva/.rvm/gems/ruby-2.1.2/gems/bunny-1.2.1/lib/bunny/cruby/socket.rb:41:in `read_nonblock'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/bunny-1.2.1/lib/bunny/cruby/socket.rb:41:in `block in read_fully'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/bunny-1.2.1/lib/bunny/cruby/socket.rb:40:in `loop'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/bunny-1.2.1/lib/bunny/cruby/socket.rb:40:in `read_fully'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/bunny-1.2.1/lib/bunny/transport.rb:199:in `read_next_frame'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/bunny-1.2.1/lib/bunny/reader_loop.rb:63:in `run_once'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/bunny-1.2.1/lib/bunny/reader_loop.rb:33:in `block in run_loop'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/bunny-1.2.1/lib/bunny/reader_loop.rb:30:in `loop'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/bunny-1.2.1/lib/bunny/reader_loop.rb:30:in `run_loop'
E, [2014-07-14T17:53:58.270357 #25232] ERROR -- : no such frame (ArgumentError)
/home/yuva/.rvm/gems/ruby-2.1.2/gems/binding_of_caller-0.7.2/lib/binding_of_caller/mri2.rb:25:in `frame_binding'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/binding_of_caller-0.7.2/lib/binding_of_caller/mri2.rb:25:in `block (2 levels) in callers'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/binding_of_caller-0.7.2/lib/binding_of_caller/mri2.rb:23:in `loop'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/binding_of_caller-0.7.2/lib/binding_of_caller/mri2.rb:23:in `block in callers'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/binding_of_caller-0.7.2/lib/binding_of_caller/mri2.rb:21:in `open'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/binding_of_caller-0.7.2/lib/binding_of_caller/mri2.rb:21:in `callers'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/better_errors-1.1.0/lib/better_errors/core_ext/exception.rb:9:in `block in <class:Exception>'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/bunny-1.2.1/lib/bunny/cruby/socket.rb:41:in `read_nonblock'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/bunny-1.2.1/lib/bunny/cruby/socket.rb:41:in `block in read_fully'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/bunny-1.2.1/lib/bunny/cruby/socket.rb:40:in `loop'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/bunny-1.2.1/lib/bunny/cruby/socket.rb:40:in `read_fully'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/bunny-1.2.1/lib/bunny/transport.rb:199:in `read_next_frame'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/bunny-1.2.1/lib/bunny/reader_loop.rb:63:in `run_once'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/bunny-1.2.1/lib/bunny/reader_loop.rb:33:in `block in run_loop'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/bunny-1.2.1/lib/bunny/reader_loop.rb:30:in `loop'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/bunny-1.2.1/lib/bunny/reader_loop.rb:30:in `run_loop'
E, [2014-07-14T17:53:58.270763 #25232] ERROR -- : stream closed (IOError)

E, [2014-07-14T17:53:58.270976 #25232] ERROR -- : failed to allocate memory (NoMemoryError)

E, [2014-07-14T17:53:58.271086 #25232] ERROR -- : stack level too deep (SystemStackError)

E, [2014-07-14T17:53:58.282903 #25232] ERROR -- : Value will be available once the pipeline executes. (Redis::FutureNotReady)

E, [2014-07-14T17:53:58.306609 #25232] ERROR -- : no such frame (ArgumentError)
/home/yuva/.rvm/gems/ruby-2.1.2/gems/binding_of_caller-0.7.2/lib/binding_of_caller/mri2.rb:25:in `frame_binding'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/binding_of_caller-0.7.2/lib/binding_of_caller/mri2.rb:25:in `block (2 levels) in callers'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/binding_of_caller-0.7.2/lib/binding_of_caller/mri2.rb:23:in `loop'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/binding_of_caller-0.7.2/lib/binding_of_caller/mri2.rb:23:in `block in callers'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/binding_of_caller-0.7.2/lib/binding_of_caller/mri2.rb:21:in `open'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/binding_of_caller-0.7.2/lib/binding_of_caller/mri2.rb:21:in `callers'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/better_errors-1.1.0/lib/better_errors/core_ext/exception.rb:9:in `block in <class:Exception>'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/bunny-1.2.1/lib/bunny/cruby/socket.rb:41:in `read_nonblock'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/bunny-1.2.1/lib/bunny/cruby/socket.rb:41:in `block in read_fully'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/bunny-1.2.1/lib/bunny/cruby/socket.rb:40:in `loop'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/bunny-1.2.1/lib/bunny/cruby/socket.rb:40:in `read_fully'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/bunny-1.2.1/lib/bunny/transport.rb:199:in `read_next_frame'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/bunny-1.2.1/lib/bunny/reader_loop.rb:63:in `run_once'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/bunny-1.2.1/lib/bunny/reader_loop.rb:33:in `block in run_loop'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/bunny-1.2.1/lib/bunny/reader_loop.rb:30:in `loop'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/bunny-1.2.1/lib/bunny/reader_loop.rb:30:in `run_loop'
E, [2014-07-14T17:53:58.306768 #25232] ERROR -- : Resource temporarily unavailable - read would block (IO::EAGAINWaitReadable)
/home/yuva/.rvm/gems/ruby-2.1.2/gems/bunny-1.2.1/lib/bunny/cruby/socket.rb:41:in `read_nonblock'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/bunny-1.2.1/lib/bunny/cruby/socket.rb:41:in `block in read_fully'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/bunny-1.2.1/lib/bunny/cruby/socket.rb:40:in `loop'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/bunny-1.2.1/lib/bunny/cruby/socket.rb:40:in `read_fully'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/bunny-1.2.1/lib/bunny/transport.rb:199:in `read_next_frame'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/bunny-1.2.1/lib/bunny/reader_loop.rb:63:in `run_once'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/bunny-1.2.1/lib/bunny/reader_loop.rb:33:in `block in run_loop'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/bunny-1.2.1/lib/bunny/reader_loop.rb:30:in `loop'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/bunny-1.2.1/lib/bunny/reader_loop.rb:30:in `run_loop'
E, [2014-07-14T17:53:58.306897 #25232] ERROR -- : Resource temporarily unavailable - read would block (IO::EAGAINWaitReadable)
/home/yuva/.rvm/gems/ruby-2.1.2/gems/bunny-1.2.1/lib/bunny/cruby/socket.rb:41:in `read_nonblock'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/bunny-1.2.1/lib/bunny/cruby/socket.rb:41:in `block in read_fully'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/bunny-1.2.1/lib/bunny/cruby/socket.rb:40:in `loop'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/bunny-1.2.1/lib/bunny/cruby/socket.rb:40:in `read_fully'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/bunny-1.2.1/lib/bunny/transport.rb:199:in `read_next_frame'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/bunny-1.2.1/lib/bunny/reader_loop.rb:63:in `run_once'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/bunny-1.2.1/lib/bunny/reader_loop.rb:33:in `block in run_loop'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/bunny-1.2.1/lib/bunny/reader_loop.rb:30:in `loop'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/bunny-1.2.1/lib/bunny/reader_loop.rb:30:in `run_loop'
E, [2014-07-14T17:53:58.307062 #25232] ERROR -- : no such frame (ArgumentError)
/home/yuva/.rvm/gems/ruby-2.1.2/gems/binding_of_caller-0.7.2/lib/binding_of_caller/mri2.rb:25:in `frame_binding'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/binding_of_caller-0.7.2/lib/binding_of_caller/mri2.rb:25:in `block (2 levels) in callers'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/binding_of_caller-0.7.2/lib/binding_of_caller/mri2.rb:23:in `loop'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/binding_of_caller-0.7.2/lib/binding_of_caller/mri2.rb:23:in `block in callers'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/binding_of_caller-0.7.2/lib/binding_of_caller/mri2.rb:21:in `open'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/binding_of_caller-0.7.2/lib/binding_of_caller/mri2.rb:21:in `callers'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/better_errors-1.1.0/lib/better_errors/core_ext/exception.rb:9:in `block in <class:Exception>'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/bunny-1.2.1/lib/bunny/cruby/socket.rb:41:in `read_nonblock'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/bunny-1.2.1/lib/bunny/cruby/socket.rb:41:in `block in read_fully'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/bunny-1.2.1/lib/bunny/cruby/socket.rb:40:in `loop'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/bunny-1.2.1/lib/bunny/cruby/socket.rb:40:in `read_fully'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/bunny-1.2.1/lib/bunny/transport.rb:199:in `read_next_frame'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/bunny-1.2.1/lib/bunny/reader_loop.rb:63:in `run_once'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/bunny-1.2.1/lib/bunny/reader_loop.rb:33:in `block in run_loop'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/bunny-1.2.1/lib/bunny/reader_loop.rb:30:in `loop'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/bunny-1.2.1/lib/bunny/reader_loop.rb:30:in `run_loop'
E, [2014-07-14T17:53:58.307138 #25232] ERROR -- : Resource temporarily unavailable - read would block (IO::EAGAINWaitReadable)
/home/yuva/.rvm/gems/ruby-2.1.2/gems/bunny-1.2.1/lib/bunny/cruby/socket.rb:41:in `read_nonblock'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/bunny-1.2.1/lib/bunny/cruby/socket.rb:41:in `block in read_fully'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/bunny-1.2.1/lib/bunny/cruby/socket.rb:40:in `loop'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/bunny-1.2.1/lib/bunny/cruby/socket.rb:40:in `read_fully'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/bunny-1.2.1/lib/bunny/transport.rb:199:in `read_next_frame'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/bunny-1.2.1/lib/bunny/reader_loop.rb:63:in `run_once'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/bunny-1.2.1/lib/bunny/reader_loop.rb:33:in `block in run_loop'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/bunny-1.2.1/lib/bunny/reader_loop.rb:30:in `loop'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/bunny-1.2.1/lib/bunny/reader_loop.rb:30:in `run_loop'
E, [2014-07-14T17:53:58.307242 #25232] ERROR -- : no such frame (ArgumentError)
/home/yuva/.rvm/gems/ruby-2.1.2/gems/binding_of_caller-0.7.2/lib/binding_of_caller/mri2.rb:25:in `frame_binding'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/binding_of_caller-0.7.2/lib/binding_of_caller/mri2.rb:25:in `block (2 levels) in callers'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/binding_of_caller-0.7.2/lib/binding_of_caller/mri2.rb:23:in `loop'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/binding_of_caller-0.7.2/lib/binding_of_caller/mri2.rb:23:in `block in callers'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/binding_of_caller-0.7.2/lib/binding_of_caller/mri2.rb:21:in `open'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/binding_of_caller-0.7.2/lib/binding_of_caller/mri2.rb:21:in `callers'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/better_errors-1.1.0/lib/better_errors/core_ext/exception.rb:9:in `block in <class:Exception>'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/bunny-1.2.1/lib/bunny/cruby/socket.rb:41:in `read_nonblock'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/bunny-1.2.1/lib/bunny/cruby/socket.rb:41:in `block in read_fully'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/bunny-1.2.1/lib/bunny/cruby/socket.rb:40:in `loop'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/bunny-1.2.1/lib/bunny/cruby/socket.rb:40:in `read_fully'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/bunny-1.2.1/lib/bunny/transport.rb:199:in `read_next_frame'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/bunny-1.2.1/lib/bunny/reader_loop.rb:63:in `run_once'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/bunny-1.2.1/lib/bunny/reader_loop.rb:33:in `block in run_loop'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/bunny-1.2.1/lib/bunny/reader_loop.rb:30:in `loop'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/bunny-1.2.1/lib/bunny/reader_loop.rb:30:in `run_loop'
E, [2014-07-14T17:53:58.307308 #25232] ERROR -- : no such frame (ArgumentError)
/home/yuva/.rvm/gems/ruby-2.1.2/gems/binding_of_caller-0.7.2/lib/binding_of_caller/mri2.rb:25:in `frame_binding'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/binding_of_caller-0.7.2/lib/binding_of_caller/mri2.rb:25:in `block (2 levels) in callers'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/binding_of_caller-0.7.2/lib/binding_of_caller/mri2.rb:23:in `loop'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/binding_of_caller-0.7.2/lib/binding_of_caller/mri2.rb:23:in `block in callers'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/binding_of_caller-0.7.2/lib/binding_of_caller/mri2.rb:21:in `open'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/binding_of_caller-0.7.2/lib/binding_of_caller/mri2.rb:21:in `callers'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/better_errors-1.1.0/lib/better_errors/core_ext/exception.rb:9:in `block in <class:Exception>'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/bunny-1.2.1/lib/bunny/cruby/socket.rb:41:in `read_nonblock'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/bunny-1.2.1/lib/bunny/cruby/socket.rb:41:in `block in read_fully'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/bunny-1.2.1/lib/bunny/cruby/socket.rb:40:in `loop'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/bunny-1.2.1/lib/bunny/cruby/socket.rb:40:in `read_fully'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/bunny-1.2.1/lib/bunny/transport.rb:199:in `read_next_frame'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/bunny-1.2.1/lib/bunny/reader_loop.rb:63:in `run_once'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/bunny-1.2.1/lib/bunny/reader_loop.rb:33:in `block in run_loop'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/bunny-1.2.1/lib/bunny/reader_loop.rb:30:in `loop'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/bunny-1.2.1/lib/bunny/reader_loop.rb:30:in `run_loop'
E, [2014-07-14T17:53:58.307413 #25232] ERROR -- : Resource temporarily unavailable - read would block (IO::EAGAINWaitReadable)
/home/yuva/.rvm/gems/ruby-2.1.2/gems/bunny-1.2.1/lib/bunny/cruby/socket.rb:41:in `read_nonblock'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/bunny-1.2.1/lib/bunny/cruby/socket.rb:41:in `block in read_fully'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/bunny-1.2.1/lib/bunny/cruby/socket.rb:40:in `loop'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/bunny-1.2.1/lib/bunny/cruby/socket.rb:40:in `read_fully'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/bunny-1.2.1/lib/bunny/transport.rb:199:in `read_next_frame'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/bunny-1.2.1/lib/bunny/reader_loop.rb:63:in `run_once'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/bunny-1.2.1/lib/bunny/reader_loop.rb:33:in `block in run_loop'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/bunny-1.2.1/lib/bunny/reader_loop.rb:30:in `loop'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/bunny-1.2.1/lib/bunny/reader_loop.rb:30:in `run_loop'
E, [2014-07-14T17:53:58.307778 #25232] ERROR -- : no such frame (ArgumentError)
/home/yuva/.rvm/gems/ruby-2.1.2/gems/binding_of_caller-0.7.2/lib/binding_of_caller/mri2.rb:25:in `frame_binding'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/binding_of_caller-0.7.2/lib/binding_of_caller/mri2.rb:25:in `block (2 levels) in callers'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/binding_of_caller-0.7.2/lib/binding_of_caller/mri2.rb:23:in `loop'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/binding_of_caller-0.7.2/lib/binding_of_caller/mri2.rb:23:in `block in callers'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/binding_of_caller-0.7.2/lib/binding_of_caller/mri2.rb:21:in `open'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/binding_of_caller-0.7.2/lib/binding_of_caller/mri2.rb:21:in `callers'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/better_errors-1.1.0/lib/better_errors/core_ext/exception.rb:9:in `block in <class:Exception>'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/bunny-1.2.1/lib/bunny/cruby/socket.rb:41:in `read_nonblock'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/bunny-1.2.1/lib/bunny/cruby/socket.rb:41:in `block in read_fully'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/bunny-1.2.1/lib/bunny/cruby/socket.rb:40:in `loop'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/bunny-1.2.1/lib/bunny/cruby/socket.rb:40:in `read_fully'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/bunny-1.2.1/lib/bunny/transport.rb:199:in `read_next_frame'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/bunny-1.2.1/lib/bunny/reader_loop.rb:63:in `run_once'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/bunny-1.2.1/lib/bunny/reader_loop.rb:33:in `block in run_loop'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/bunny-1.2.1/lib/bunny/reader_loop.rb:30:in `loop'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/bunny-1.2.1/lib/bunny/reader_loop.rb:30:in `run_loop'
E, [2014-07-14T17:53:58.307886 #25232] ERROR -- : Resource temporarily unavailable - read would block (IO::EAGAINWaitReadable)
/home/yuva/.rvm/gems/ruby-2.1.2/gems/bunny-1.2.1/lib/bunny/cruby/socket.rb:41:in `read_nonblock'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/bunny-1.2.1/lib/bunny/cruby/socket.rb:41:in `block in read_fully'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/bunny-1.2.1/lib/bunny/cruby/socket.rb:40:in `loop'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/bunny-1.2.1/lib/bunny/cruby/socket.rb:40:in `read_fully'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/bunny-1.2.1/lib/bunny/transport.rb:199:in `read_next_frame'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/bunny-1.2.1/lib/bunny/reader_loop.rb:63:in `run_once'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/bunny-1.2.1/lib/bunny/reader_loop.rb:33:in `block in run_loop'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/bunny-1.2.1/lib/bunny/reader_loop.rb:30:in `loop'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/bunny-1.2.1/lib/bunny/reader_loop.rb:30:in `run_loop'
E, [2014-07-14T17:53:58.307968 #25232] ERROR -- : Resource temporarily unavailable - read would block (IO::EAGAINWaitReadable)
/home/yuva/.rvm/gems/ruby-2.1.2/gems/bunny-1.2.1/lib/bunny/cruby/socket.rb:41:in `read_nonblock'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/bunny-1.2.1/lib/bunny/cruby/socket.rb:41:in `block in read_fully'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/bunny-1.2.1/lib/bunny/cruby/socket.rb:40:in `loop'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/bunny-1.2.1/lib/bunny/cruby/socket.rb:40:in `read_fully'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/bunny-1.2.1/lib/bunny/transport.rb:199:in `read_next_frame'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/bunny-1.2.1/lib/bunny/reader_loop.rb:63:in `run_once'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/bunny-1.2.1/lib/bunny/reader_loop.rb:33:in `block in run_loop'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/bunny-1.2.1/lib/bunny/reader_loop.rb:30:in `loop'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/bunny-1.2.1/lib/bunny/reader_loop.rb:30:in `run_loop'
E, [2014-07-14T17:53:58.308202 #25232] ERROR -- : no such frame (ArgumentError)
/home/yuva/.rvm/gems/ruby-2.1.2/gems/binding_of_caller-0.7.2/lib/binding_of_caller/mri2.rb:25:in `frame_binding'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/binding_of_caller-0.7.2/lib/binding_of_caller/mri2.rb:25:in `block (2 levels) in callers'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/binding_of_caller-0.7.2/lib/binding_of_caller/mri2.rb:23:in `loop'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/binding_of_caller-0.7.2/lib/binding_of_caller/mri2.rb:23:in `block in callers'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/binding_of_caller-0.7.2/lib/binding_of_caller/mri2.rb:21:in `open'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/binding_of_caller-0.7.2/lib/binding_of_caller/mri2.rb:21:in `callers'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/better_errors-1.1.0/lib/better_errors/core_ext/exception.rb:9:in `block in <class:Exception>'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/bunny-1.2.1/lib/bunny/cruby/socket.rb:41:in `read_nonblock'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/bunny-1.2.1/lib/bunny/cruby/socket.rb:41:in `block in read_fully'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/bunny-1.2.1/lib/bunny/cruby/socket.rb:40:in `loop'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/bunny-1.2.1/lib/bunny/cruby/socket.rb:40:in `read_fully'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/bunny-1.2.1/lib/bunny/transport.rb:199:in `read_next_frame'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/bunny-1.2.1/lib/bunny/reader_loop.rb:63:in `run_once'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/bunny-1.2.1/lib/bunny/reader_loop.rb:33:in `block in run_loop'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/bunny-1.2.1/lib/bunny/reader_loop.rb:30:in `loop'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/bunny-1.2.1/lib/bunny/reader_loop.rb:30:in `run_loop'
E, [2014-07-14T17:53:58.308385 #25232] ERROR -- : Resource temporarily unavailable - read would block (IO::EAGAINWaitReadable)
/home/yuva/.rvm/gems/ruby-2.1.2/gems/bunny-1.2.1/lib/bunny/cruby/socket.rb:41:in `read_nonblock'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/bunny-1.2.1/lib/bunny/cruby/socket.rb:41:in `block in read_fully'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/bunny-1.2.1/lib/bunny/cruby/socket.rb:40:in `loop'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/bunny-1.2.1/lib/bunny/cruby/socket.rb:40:in `read_fully'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/bunny-1.2.1/lib/bunny/transport.rb:199:in `read_next_frame'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/bunny-1.2.1/lib/bunny/reader_loop.rb:63:in `run_once'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/bunny-1.2.1/lib/bunny/reader_loop.rb:33:in `block in run_loop'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/bunny-1.2.1/lib/bunny/reader_loop.rb:30:in `loop'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/bunny-1.2.1/lib/bunny/reader_loop.rb:30:in `run_loop'
E, [2014-07-14T17:53:58.308603 #25232] ERROR -- : no such frame (ArgumentError)
/home/yuva/.rvm/gems/ruby-2.1.2/gems/binding_of_caller-0.7.2/lib/binding_of_caller/mri2.rb:25:in `frame_binding'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/binding_of_caller-0.7.2/lib/binding_of_caller/mri2.rb:25:in `block (2 levels) in callers'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/binding_of_caller-0.7.2/lib/binding_of_caller/mri2.rb:23:in `loop'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/binding_of_caller-0.7.2/lib/binding_of_caller/mri2.rb:23:in `block in callers'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/binding_of_caller-0.7.2/lib/binding_of_caller/mri2.rb:21:in `open'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/binding_of_caller-0.7.2/lib/binding_of_caller/mri2.rb:21:in `callers'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/better_errors-1.1.0/lib/better_errors/core_ext/exception.rb:9:in `block in <class:Exception>'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/bunny-1.2.1/lib/bunny/cruby/socket.rb:41:in `read_nonblock'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/bunny-1.2.1/lib/bunny/cruby/socket.rb:41:in `block in read_fully'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/bunny-1.2.1/lib/bunny/cruby/socket.rb:40:in `loop'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/bunny-1.2.1/lib/bunny/cruby/socket.rb:40:in `read_fully'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/bunny-1.2.1/lib/bunny/transport.rb:199:in `read_next_frame'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/bunny-1.2.1/lib/bunny/reader_loop.rb:63:in `run_once'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/bunny-1.2.1/lib/bunny/reader_loop.rb:33:in `block in run_loop'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/bunny-1.2.1/lib/bunny/reader_loop.rb:30:in `loop'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/bunny-1.2.1/lib/bunny/reader_loop.rb:30:in `run_loop'
E, [2014-07-14T17:53:58.309189 #25232] ERROR -- : no such frame (ArgumentError)
/home/yuva/.rvm/gems/ruby-2.1.2/gems/binding_of_caller-0.7.2/lib/binding_of_caller/mri2.rb:25:in `frame_binding'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/binding_of_caller-0.7.2/lib/binding_of_caller/mri2.rb:25:in `block (2 levels) in callers'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/binding_of_caller-0.7.2/lib/binding_of_caller/mri2.rb:23:in `loop'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/binding_of_caller-0.7.2/lib/binding_of_caller/mri2.rb:23:in `block in callers'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/binding_of_caller-0.7.2/lib/binding_of_caller/mri2.rb:21:in `open'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/binding_of_caller-0.7.2/lib/binding_of_caller/mri2.rb:21:in `callers'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/better_errors-1.1.0/lib/better_errors/core_ext/exception.rb:9:in `block in <class:Exception>'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:55:in `join'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:55:in `block in join'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:55:in `each'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:55:in `join'
/home/yuva/Desktop/arvind/peatio/peatio-master/lib/daemons/amqp_daemon.rb:80:in `<top (required)>'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/daemons-1.1.9/lib/daemons/application.rb:203:in `load'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/daemons-1.1.9/lib/daemons/application.rb:203:in `start_load'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/daemons-1.1.9/lib/daemons/application.rb:298:in `start'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/daemons-1.1.9/lib/daemons/controller.rb:70:in `run'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/daemons-1.1.9/lib/daemons.rb:147:in `block in run'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/daemons-1.1.9/lib/daemons/cmdline.rb:109:in `call'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/daemons-1.1.9/lib/daemons/cmdline.rb:109:in `catch_exceptions'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/daemons-1.1.9/lib/daemons.rb:146:in `run'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/daemons-rails-1.2.1/lib/daemons/rails.rb:16:in `run'
lib/daemons/matching_ctl:11:in `<main>'
E, [2014-07-14T17:53:58.309291 #25232] ERROR -- : no such frame (ArgumentError)
/home/yuva/.rvm/gems/ruby-2.1.2/gems/binding_of_caller-0.7.2/lib/binding_of_caller/mri2.rb:25:in `frame_binding'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/binding_of_caller-0.7.2/lib/binding_of_caller/mri2.rb:25:in `block (2 levels) in callers'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/binding_of_caller-0.7.2/lib/binding_of_caller/mri2.rb:23:in `loop'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/binding_of_caller-0.7.2/lib/binding_of_caller/mri2.rb:23:in `block in callers'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/binding_of_caller-0.7.2/lib/binding_of_caller/mri2.rb:21:in `open'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/binding_of_caller-0.7.2/lib/binding_of_caller/mri2.rb:21:in `callers'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/better_errors-1.1.0/lib/better_errors/core_ext/exception.rb:9:in `block in <class:Exception>'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/bunny-1.2.1/lib/bunny/reader_loop.rb:94:in `raise'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/bunny-1.2.1/lib/bunny/reader_loop.rb:94:in `raise'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/bunny-1.2.1/lib/bunny/session.rb:796:in `maybe_shutdown_reader_loop'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/bunny-1.2.1/lib/bunny/session.rb:705:in `clean_up_on_shutdown'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/bunny-1.2.1/lib/bunny/session.rb:698:in `clean_up_and_fail_on_connection_close!'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/bunny-1.2.1/lib/bunny/session.rb:533:in `handle_frame'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/bunny-1.2.1/lib/bunny/reader_loop.rb:81:in `run_once'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/bunny-1.2.1/lib/bunny/reader_loop.rb:33:in `block in run_loop'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/bunny-1.2.1/lib/bunny/reader_loop.rb:30:in `loop'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/bunny-1.2.1/lib/bunny/reader_loop.rb:30:in `run_loop'
E, [2014-07-14T17:53:58.309372 #25232] ERROR -- : Bunny::ShutdownSignal (Bunny::ShutdownSignal)
/home/yuva/.rvm/gems/ruby-2.1.2/gems/bunny-1.2.1/lib/bunny/reader_loop.rb:94:in `raise'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/bunny-1.2.1/lib/bunny/reader_loop.rb:94:in `raise'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/bunny-1.2.1/lib/bunny/session.rb:796:in `maybe_shutdown_reader_loop'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/bunny-1.2.1/lib/bunny/session.rb:705:in `clean_up_on_shutdown'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/bunny-1.2.1/lib/bunny/session.rb:698:in `clean_up_and_fail_on_connection_close!'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/bunny-1.2.1/lib/bunny/session.rb:533:in `handle_frame'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/bunny-1.2.1/lib/bunny/reader_loop.rb:81:in `run_once'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/bunny-1.2.1/lib/bunny/reader_loop.rb:33:in `block in run_loop'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/bunny-1.2.1/lib/bunny/reader_loop.rb:30:in `loop'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/bunny-1.2.1/lib/bunny/reader_loop.rb:30:in `run_loop'
E, [2014-07-14T17:53:58.309425 #25232] ERROR -- : Connection-level error: CONNECTION_FORCED - broker forced connection closure with reason 'shutdown' (Bunny::ConnectionForced)
/home/yuva/.rvm/gems/ruby-2.1.2/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:55:in `join'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:55:in `block in join'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:55:in `each'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:55:in `join'
/home/yuva/Desktop/arvind/peatio/peatio-master/lib/daemons/amqp_daemon.rb:80:in `<top (required)>'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/daemons-1.1.9/lib/daemons/application.rb:203:in `load'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/daemons-1.1.9/lib/daemons/application.rb:203:in `start_load'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/daemons-1.1.9/lib/daemons/application.rb:298:in `start'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/daemons-1.1.9/lib/daemons/controller.rb:70:in `run'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/daemons-1.1.9/lib/daemons.rb:147:in `block in run'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/daemons-1.1.9/lib/daemons/cmdline.rb:109:in `call'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/daemons-1.1.9/lib/daemons/cmdline.rb:109:in `catch_exceptions'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/daemons-1.1.9/lib/daemons.rb:146:in `run'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/daemons-rails-1.2.1/lib/daemons/rails.rb:16:in `run'
lib/daemons/matching_ctl:11:in `<main>'
E, [2014-07-14T17:53:58.309508 #25232] ERROR -- : no such frame (ArgumentError)
/home/yuva/.rvm/gems/ruby-2.1.2/gems/binding_of_caller-0.7.2/lib/binding_of_caller/mri2.rb:25:in `frame_binding'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/binding_of_caller-0.7.2/lib/binding_of_caller/mri2.rb:25:in `block (2 levels) in callers'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/binding_of_caller-0.7.2/lib/binding_of_caller/mri2.rb:23:in `loop'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/binding_of_caller-0.7.2/lib/binding_of_caller/mri2.rb:23:in `block in callers'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/binding_of_caller-0.7.2/lib/binding_of_caller/mri2.rb:21:in `open'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/binding_of_caller-0.7.2/lib/binding_of_caller/mri2.rb:21:in `callers'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/better_errors-1.1.0/lib/better_errors/core_ext/exception.rb:9:in `block in <class:Exception>'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/daemons-1.1.9/lib/daemons/pidfile.rb:107:in `gets'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/daemons-1.1.9/lib/daemons/pidfile.rb:107:in `block in pid'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/daemons-1.1.9/lib/daemons/pidfile.rb:106:in `open'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/daemons-1.1.9/lib/daemons/pidfile.rb:106:in `pid'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/daemons-1.1.9/lib/daemons/pidfile.rb:101:in `cleanup'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/daemons-1.1.9/lib/daemons/application.rb:160:in `block in start_load'
E, [2014-07-14T17:53:58.309606 #25232] ERROR -- : no such frame (ArgumentError)
/home/yuva/.rvm/gems/ruby-2.1.2/gems/binding_of_caller-0.7.2/lib/binding_of_caller/mri2.rb:25:in `frame_binding'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/binding_of_caller-0.7.2/lib/binding_of_caller/mri2.rb:25:in `block (2 levels) in callers'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/binding_of_caller-0.7.2/lib/binding_of_caller/mri2.rb:23:in `loop'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/binding_of_caller-0.7.2/lib/binding_of_caller/mri2.rb:23:in `block in callers'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/binding_of_caller-0.7.2/lib/binding_of_caller/mri2.rb:21:in `open'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/binding_of_caller-0.7.2/lib/binding_of_caller/mri2.rb:21:in `callers'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/better_errors-1.1.0/lib/better_errors/core_ext/exception.rb:9:in `block in <class:Exception>'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/bunny-1.2.1/lib/bunny/reader_loop.rb:94:in `raise'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/bunny-1.2.1/lib/bunny/reader_loop.rb:94:in `raise'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/bunny-1.2.1/lib/bunny/session.rb:796:in `maybe_shutdown_reader_loop'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/bunny-1.2.1/lib/bunny/session.rb:705:in `clean_up_on_shutdown'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/bunny-1.2.1/lib/bunny/session.rb:698:in `clean_up_and_fail_on_connection_close!'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/bunny-1.2.1/lib/bunny/session.rb:533:in `handle_frame'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/bunny-1.2.1/lib/bunny/reader_loop.rb:81:in `run_once'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/bunny-1.2.1/lib/bunny/reader_loop.rb:33:in `block in run_loop'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/bunny-1.2.1/lib/bunny/reader_loop.rb:30:in `loop'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/bunny-1.2.1/lib/bunny/reader_loop.rb:30:in `run_loop'
E, [2014-07-14T17:53:58.309691 #25232] ERROR -- : Bunny::ShutdownSignal (Bunny::ShutdownSignal)
/home/yuva/.rvm/gems/ruby-2.1.2/gems/bunny-1.2.1/lib/bunny/reader_loop.rb:94:in `raise'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/bunny-1.2.1/lib/bunny/reader_loop.rb:94:in `raise'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/bunny-1.2.1/lib/bunny/session.rb:796:in `maybe_shutdown_reader_loop'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/bunny-1.2.1/lib/bunny/session.rb:705:in `clean_up_on_shutdown'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/bunny-1.2.1/lib/bunny/session.rb:698:in `clean_up_and_fail_on_connection_close!'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/bunny-1.2.1/lib/bunny/session.rb:533:in `handle_frame'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/bunny-1.2.1/lib/bunny/reader_loop.rb:81:in `run_once'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/bunny-1.2.1/lib/bunny/reader_loop.rb:33:in `block in run_loop'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/bunny-1.2.1/lib/bunny/reader_loop.rb:30:in `loop'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/bunny-1.2.1/lib/bunny/reader_loop.rb:30:in `run_loop'
E, [2014-07-14T17:53:58.309739 #25232] ERROR -- : Connection-level error: CONNECTION_FORCED - broker forced connection closure with reason 'shutdown' (Bunny::ConnectionForced)
/home/yuva/.rvm/gems/ruby-2.1.2/gems/daemons-1.1.9/lib/daemons/pidfile.rb:107:in `gets'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/daemons-1.1.9/lib/daemons/pidfile.rb:107:in `block in pid'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/daemons-1.1.9/lib/daemons/pidfile.rb:106:in `open'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/daemons-1.1.9/lib/daemons/pidfile.rb:106:in `pid'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/daemons-1.1.9/lib/daemons/pidfile.rb:101:in `cleanup'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/daemons-1.1.9/lib/daemons/application.rb:160:in `block in start_load'
E, [2014-07-14T17:53:58.323944 #25232] ERROR -- : Resource temporarily unavailable - read would block (IO::EAGAINWaitReadable)
/home/yuva/.rvm/gems/ruby-2.1.2/gems/bunny-1.2.1/lib/bunny/cruby/socket.rb:41:in `read_nonblock'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/bunny-1.2.1/lib/bunny/cruby/socket.rb:41:in `block in read_fully'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/bunny-1.2.1/lib/bunny/cruby/socket.rb:40:in `loop'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/bunny-1.2.1/lib/bunny/cruby/socket.rb:40:in `read_fully'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/bunny-1.2.1/lib/bunny/transport.rb:199:in `read_next_frame'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/bunny-1.2.1/lib/bunny/session.rb:990:in `open_connection'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/bunny-1.2.1/lib/bunny/session.rb:255:in `start'
/home/yuva/Desktop/arvind/peatio/peatio-master/lib/daemons/amqp_daemon.rb:17:in `<top (required)>'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/daemons-1.1.9/lib/daemons/application.rb:203:in `load'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/daemons-1.1.9/lib/daemons/application.rb:203:in `start_load'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/daemons-1.1.9/lib/daemons/application.rb:298:in `start'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/daemons-1.1.9/lib/daemons/controller.rb:70:in `run'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/daemons-1.1.9/lib/daemons.rb:147:in `block in run'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/daemons-1.1.9/lib/daemons/cmdline.rb:109:in `call'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/daemons-1.1.9/lib/daemons/cmdline.rb:109:in `catch_exceptions'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/daemons-1.1.9/lib/daemons.rb:146:in `run'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/daemons-rails-1.2.1/lib/daemons/rails.rb:16:in `run'
lib/daemons/matching_ctl:11:in `<main>'
E, [2014-07-14T17:53:58.324087 #25232] ERROR -- : no such frame (ArgumentError)
/home/yuva/.rvm/gems/ruby-2.1.2/gems/binding_of_caller-0.7.2/lib/binding_of_caller/mri2.rb:25:in `frame_binding'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/binding_of_caller-0.7.2/lib/binding_of_caller/mri2.rb:25:in `block (2 levels) in callers'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/binding_of_caller-0.7.2/lib/binding_of_caller/mri2.rb:23:in `loop'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/binding_of_caller-0.7.2/lib/binding_of_caller/mri2.rb:23:in `block in callers'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/binding_of_caller-0.7.2/lib/binding_of_caller/mri2.rb:21:in `open'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/binding_of_caller-0.7.2/lib/binding_of_caller/mri2.rb:21:in `callers'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/better_errors-1.1.0/lib/better_errors/core_ext/exception.rb:9:in `block in <class:Exception>'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/bunny-1.2.1/lib/bunny/cruby/socket.rb:41:in `read_nonblock'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/bunny-1.2.1/lib/bunny/cruby/socket.rb:41:in `block in read_fully'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/bunny-1.2.1/lib/bunny/cruby/socket.rb:40:in `loop'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/bunny-1.2.1/lib/bunny/cruby/socket.rb:40:in `read_fully'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/bunny-1.2.1/lib/bunny/transport.rb:199:in `read_next_frame'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/bunny-1.2.1/lib/bunny/session.rb:923:in `init_connection'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/bunny-1.2.1/lib/bunny/session.rb:254:in `start'
/home/yuva/Desktop/arvind/peatio/peatio-master/lib/daemons/amqp_daemon.rb:17:in `<top (required)>'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/daemons-1.1.9/lib/daemons/application.rb:203:in `load'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/daemons-1.1.9/lib/daemons/application.rb:203:in `start_load'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/daemons-1.1.9/lib/daemons/application.rb:298:in `start'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/daemons-1.1.9/lib/daemons/controller.rb:70:in `run'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/daemons-1.1.9/lib/daemons.rb:147:in `block in run'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/daemons-1.1.9/lib/daemons/cmdline.rb:109:in `call'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/daemons-1.1.9/lib/daemons/cmdline.rb:109:in `catch_exceptions'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/daemons-1.1.9/lib/daemons.rb:146:in `run'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/daemons-rails-1.2.1/lib/daemons/rails.rb:16:in `run'
lib/daemons/matching_ctl:11:in `<main>'
E, [2014-07-14T17:53:58.324174 #25232] ERROR -- : Resource temporarily unavailable - read would block (IO::EAGAINWaitReadable)
/home/yuva/.rvm/gems/ruby-2.1.2/gems/bunny-1.2.1/lib/bunny/cruby/socket.rb:41:in `read_nonblock'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/bunny-1.2.1/lib/bunny/cruby/socket.rb:41:in `block in read_fully'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/bunny-1.2.1/lib/bunny/cruby/socket.rb:40:in `loop'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/bunny-1.2.1/lib/bunny/cruby/socket.rb:40:in `read_fully'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/bunny-1.2.1/lib/bunny/transport.rb:199:in `read_next_frame'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/bunny-1.2.1/lib/bunny/reader_loop.rb:63:in `run_once'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/bunny-1.2.1/lib/bunny/reader_loop.rb:33:in `block in run_loop'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/bunny-1.2.1/lib/bunny/reader_loop.rb:30:in `loop'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/bunny-1.2.1/lib/bunny/reader_loop.rb:30:in `run_loop'
E, [2014-07-14T17:53:58.324245 #25232] ERROR -- : no such frame (ArgumentError)
/home/yuva/.rvm/gems/ruby-2.1.2/gems/binding_of_caller-0.7.2/lib/binding_of_caller/mri2.rb:25:in `frame_binding'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/binding_of_caller-0.7.2/lib/binding_of_caller/mri2.rb:25:in `block (2 levels) in callers'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/binding_of_caller-0.7.2/lib/binding_of_caller/mri2.rb:23:in `loop'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/binding_of_caller-0.7.2/lib/binding_of_caller/mri2.rb:23:in `block in callers'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/binding_of_caller-0.7.2/lib/binding_of_caller/mri2.rb:21:in `open'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/binding_of_caller-0.7.2/lib/binding_of_caller/mri2.rb:21:in `callers'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/better_errors-1.1.0/lib/better_errors/core_ext/exception.rb:9:in `block in <class:Exception>'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/bunny-1.2.1/lib/bunny/cruby/socket.rb:41:in `read_nonblock'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/bunny-1.2.1/lib/bunny/cruby/socket.rb:41:in `block in read_fully'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/bunny-1.2.1/lib/bunny/cruby/socket.rb:40:in `loop'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/bunny-1.2.1/lib/bunny/cruby/socket.rb:40:in `read_fully'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/bunny-1.2.1/lib/bunny/transport.rb:199:in `read_next_frame'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/bunny-1.2.1/lib/bunny/session.rb:990:in `open_connection'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/bunny-1.2.1/lib/bunny/session.rb:255:in `start'
/home/yuva/Desktop/arvind/peatio/peatio-master/lib/daemons/amqp_daemon.rb:17:in `<top (required)>'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/daemons-1.1.9/lib/daemons/application.rb:203:in `load'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/daemons-1.1.9/lib/daemons/application.rb:203:in `start_load'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/daemons-1.1.9/lib/daemons/application.rb:298:in `start'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/daemons-1.1.9/lib/daemons/controller.rb:70:in `run'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/daemons-1.1.9/lib/daemons.rb:147:in `block in run'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/daemons-1.1.9/lib/daemons/cmdline.rb:109:in `call'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/daemons-1.1.9/lib/daemons/cmdline.rb:109:in `catch_exceptions'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/daemons-1.1.9/lib/daemons.rb:146:in `run'
/home/yuva/.rvm/gems/ruby-2.1.2/gems/daemons-rails-1.2.1/lib/daemons/rails.rb:16:in `run'
lib/daemons/matching_ctl:11:in `<main>'
